frontend/src/layouts/new-layout.js

```diff
@@ -1,12 +1,319 @@
 import React, { useState } from 'react'
 import { useRouter } from 'next/router'
 import { isEmpty } from 'lodash'
-import { Button } from 'antd'
+import { Button, Col, Row } from 'antd'
 import localFont from 'next/font/local'
 import { DM_Sans } from 'next/font/google'
 import Image from 'next/image'
 import Footer from '../footer'
 import Login from '../modules/login/view'
+import { DownArrow, CloseIcon, MenuIcon } from '../components/icons'
+import Link from 'next/link'
+import { motion, AnimatePresence, useCycle } from 'framer-motion'
+import BookIcon from '../images/book-open.svg'
+import CaseStudiesSvg from '../images/folder.svg'
+import CapacityBuildingSvg from '../images/owl.svg'
+import HelpCenterSvg from '../images/help.svg'
+import IconCommunity from '../images/community.svg'
+import ExpertIcon from '../images/education.svg'
+import IconEvent from '../images/calendar.svg'
+import IconPartner from '../images/partners.svg'
+import IconForum from '../images/engage.svg'
+import AnalyticAndStatisticSvg from '../images/statistics.svg'
+import DataCatalogueSvg from '../images/archive.svg'
+import GlossarySvg from '../images/glossary.svg'
+import MapSvg from '../images/map.svg'
+import ExploreSvg from '../images/api.svg'
+import { useDeviceSize } from '../modules/landing/landing'
+import { MenuToggle } from './MenuToggle'
+import { Navigation } from './Navigation'
+
+const navVariants = {
+  open: { scale: 1, opacity: 1 },
+  closed: { scale: 0.95, opacity: 0 },
+}
+
+const menuItemVariants = {
+  open: (i) => ({
+    opacity: 1,
+    y: 0,
+    transition: {
+      delay: 0.2 + i * 0.1,
+      duration: 0.6,
+      ease: [0.42, 0, 0.58, 1],
+    },
+  }),
+  closed: {
+    opacity: 0,
+    y: 50,
+    transition: {
+      duration: 0.5,
+      ease: 'anticipate',
+    },
+  },
+}
+
+export const menuList = [
```

Comment:
Why would you put this data in this file?

Reply:
Fullscreen menu code was in that file thats why added it there